### PR TITLE
compute, live-migrations: fix typo and improve grammer

### PR DESCRIPTION
**Pull request by nirdothan**
Fix one typo in a manifest example that is technically incorrect
In addition, improve grammar in text to make it more concise 
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Checklist**

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: docs/compute/live_migration.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ Status:
 
 ### Migration Status
 
-The migration progress status is being reported in the VMI `VMI.status`.
+The migration progress status is qreported in the VMI `VMI.status`.
```
Comment (orelmisan) [inline docs/compute/live_migration.md:77]: s/qreported/reported

**Review by orelmisan — COMMENTED** [review]
Thank you for the PR @nirdothan.
The API fix looks good to me.
@aburdenthehand since you are a native speaker, could you please review the grammatical change?

**Comment by aburdenthehand** [comment]
/lgtm 
Thanks for the corrections @nirdothan

**Review by enp0s3 — APPROVED** [review]
@nirdothan Thank you!

**Comment by phoracek** [comment]
/approve

Raising Igor's LGTM

**Pull request merged**